appium/webdriver/extensions/android/nativekey.py

```diff
@@ -1,3 +1,18 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
 class AndroidKey:
     # Key code constant: Unknown key code.
     UNKNOWN = 0
```

Comment:
Python now (since v 3.4) implements `enum` [natively](https://docs.python.org/3/library/enum.html). We could think about implementing it for some existing classes

Reply:
Will think in another PR.